answers.txt

```diff
@@ -1,8 +1,7 @@
-﻿Number of students who modified this file: 70 + 1 = 71 students

```

Comment:
After openning the file to resolve the conflicts, I counted the number of students who had made entries and it was 85, not 71, including me.